webclient/src/hooks/useReduxEffect.tsx

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
@ZeldaZach you said we need to add something to credit the source?

Reply:
```suggestion
/** 
File is adapted from https://github.com/Qeepsake/use-redux-effect under MIT License
```